### ISSUE TITLE
env: node\r: No such file or directory

### ISSUE THREAD

**Issue by blindgaenger** [typicode/hotel#133]
I'm using version 0.5.7 on OSX and it seems there is something wrong with the line ending.

```
‣ hotel start
env: node\r: No such file or directory
```

I found the same issue in other projects (https://github.com/nwjs/nw-builder/issues/75) and it was easy to fix.

```
‣ vim `which hotel`
:set fileformat=unix
:x
```

Keep up the good work!

**Comment by typicode** [comment]
Thank you for the link :+1: and heads up.
Re-published hotel (`v0.5.8`) from a Linux env and it seems better.

Previous version was published from Windows and I didn't thought there could be an issue with line endings.

Let me know if it works for you now.

**Comment by blindgaenger** [comment]
Yep, now it works like charm. Thanks! 😃
